tests/test_geoprocessing.py

```diff
@@ -1079,13 +1079,13 @@ def test_zonal_stats_all_outside(self):
         array = numpy.fliplr(numpy.flipud(
             numpy.array(range(n*n), dtype=numpy.int32).reshape((n, n))))
         _array_to_raster(
-            array, None, raster_path, projection_epsg=4326,
-            origin=(origin_x+n, origin_y-n), pixel_size=(-1, 1))
+            array, 255, raster_path, projection_epsg=4326,

```

Comment:
This seems like it was a bug in the test: lines 1075-1076 say it's meant to test intersection with a nodata pixel, but we weren't marking that pixels' value (255) as being nodata.